packages/components/src/components/editor/browser/browser.tsx

```diff
@@ -94,6 +94,7 @@ export type Browser = {
   icon: IvyIcons;
   browser: ReturnType<typeof useBrowser>;
   header?: React.ReactNode;
+  headerControl?: React.ReactNode;
```

Comment:
why... 😥

We should not add and add and add properties. If we think like that the component will get more and more complicated! 
Beside of that case an adopter of this component need to know that the header and headerControl are both needed if he wants to add a headerControl. If not the outcome is not the same!

Why should the adapter not be able to write this Flex wrapper?!?